### DISCUSSION TITLE
Enum survey vars shared among templates

### DISCUSSION THREAD

**Discussion by danni140c** [semaphoreui/semaphore#2134] [Ideas]
Would there be any interest, in having survey variables of the enum type be shared among templates? Or the possibility, to choose to have it shared or not?

This would make it easier to maintain an enum, if you want to use the same enum for multiple templates. E.g. if I wanted to select the hosts of a playbook, by using an enum survey variable, I would currently need to maintain this enum for each template, where I want to be able to select hosts from the same set/enum.

**Comment by fiftin** [comment]
Currently it is not possible but it is good idea.

**Comment by Yuri-Lima** [comment]
Definitely a good a idea. I was wondering the same thing.
